Lagrange.Core/Message/Entity/ForwardEntity.cs

```diff
@@ -79,7 +79,7 @@ IEnumerable<Elem> IMessageEntity.PackElement()
             {
                 Text = new Text
                 {
-                    Str = null,
+                    Str = "Officially, custom names are no longer allowed, but Lagrange still requires this field to resolve the displayed name",
```

Comment:
Try to construct the same Text field as the official client

Reply:
This field is the content field of the Mention, tx has turned off its customization, but lgr detects the Mention requires that this field is not empty, I did not find the reason for this, so I put random characters in this field to ensure that lgr's detection does not make a mistake